### PR TITLE
Material Design

### PR DESCRIPTION
**Pull request by tmfksoft**
Re-themed the timings panel and made some major layout changes.
jQuery-ui CSS removed in favour of custom CSS.
jQuery-ui JS will still function fine alongside custom CSS

Live preview: http://burnett.thomas-edwards.me/timings/

### CODE REVIEW

**Comment by xZeroMCPE** [comment]
@tmfksoft Live preview aint working!.....

I'd need to see it before i can merge the PR

File: image.php [hunk 1]
```diff
@@ -5,7 +5,8 @@
 $id = basename(preg_replace('/[^0-9a-f]/', '', $_GET['id']));
 $file = "/tmp/timingsimg_" . basename($_GET['id']);
 if (!file_exists($file)) {
-	die;
+	header("Content-Type: image/png");
+	readfile("static/img/unknown_server.png");
```
Comment (aikar) [inline image.php]: re-add the exit below this

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,273 @@
+0 info it worked if it ends with ok
```
Comment (aikar) [inline npm-debug.log]: delete this file

File: static/css/timings.scss [hunk 3]
```diff
@@ -1,12 +1,13 @@
 /*
  * Aikar's Minecraft Timings Parser
  *
- * Written by Aikar <aikar@aikar.co>
+ * Written by Aikar <aikar@aikar.co> - Butchered by Fudgie <thomas@ilkotech.co.uk>
```
Comment (aikar) [inline static/css/timings.scss]: copyright headers are auto generated, remove this change and add self to Contributors file instead

File: template/header-right.php [hunk 4]
```diff
@@ -1,31 +0,0 @@
-<?php
```
Comment (aikar) [inline template/header-right.php]: please update to latest, as im sure this will conflict

File: template/index.php [hunk 5]
```diff
@@ -63,10 +72,9 @@
 </div>
 
 <div id="footer">
-    <span id="footer-left">&copy; 2013-<?=date('Y')?> Starlis LLC <a href="http://github.com/aikar/timings"
-                                                      title="Source">[source]</a> &mdash; <a
+    <span id="footer-left">&copy; 2013-<?=date('Y')?> Starlis LLC <a href="http://github.com/aikar/timings" title="Source">[source]</a> &mdash; <a
 		    href="http://aikar.co/spigot/" title="About / Donate">[about / donate]</a></span>
-	<span id="footer-right"></span>
+	<span id="footer-right">Theme bashed together by Thomas Edwards</span>
```
Comment (aikar) [inline template/index.php]: Rather more professional wording here. "Theme by Thomas Edwards"

File: template/server-info.php [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+$timingsData = TimingsMaster::getInstance();
+$tpl = Template::getInstance();
+$system = $timingsData->system;
+$motd = $timingsData->motd;
+if (is_array($motd)) {
+	$motd = implode("\n", $motd);
+}
+$version = $timingsData->version;
+if (!empty($version['value'])) {
+	$version = "(Buggy Sponge Version OPTIONAL OPTIONAL) " . $version['value'];
```
Comment (aikar) [inline template/server-info.php]: Please pull in changes I did to header-right.php that got removed.

File: npm-debug.log (hunk 2 above)
Comment (tmfksoft) [inline npm-debug.log]: Would adding this file to .gitignore suffice?

File: image.php (hunk 1 above)
Comment (tmfksoft) [inline image.php]: Back where it was originally?

File: template/header-right.php (hunk 4 above)
Comment (tmfksoft) [inline template/header-right.php]: Latest version of this file?

File: template/index.php (hunk 5 above)
Comment (tmfksoft) [inline template/index.php]: Psh :P ~ This will be changed in next PR

File: static/css/timings.scss (hunk 3 above)
Comment (tmfksoft) [inline static/css/timings.scss]: Will be sorted in next PR

File: template/header-right.php (hunk 4 above)
Comment (tmfksoft) [inline template/header-right.php]: Scratch that comment, Looks like I removed header-right and made it part of the main index.php template. Would you like me to update the index.php template with the changes in header-right.php? - I removed it due to the fact the header was no longer on the right. (I believe?)

File: image.php (hunk 1 above)
Comment (aikar) [inline image.php]: or just change this to $file = static/img/unknown and let the readfile bellow handle it.

File: npm-debug.log (hunk 2 above)
Comment (aikar) [inline npm-debug.log]: delete first then gitignore

File: template/header-right.php (hunk 4 above)
Comment (aikar) [inline template/header-right.php]: I made a change to this file for sponge ver, so just make sure you carry along my change to its new home

File: image.php (hunk 1 above)
Comment (tmfksoft) [inline image.php]: I'll go for the latter

**Pull request merged**